### PR TITLE
feat(clear): Add clearChunks() function and docs.

### PR DESCRIPTION
**Pull request by threehams**
Add clearChunks() function as an explicit way of clearing chunks which may be left over from
previous requests (or two-stage render). Add documentation around usage with other libraries which
use the same approach.

I've named it `clearChunks` to be a bit more friendly with destructuring, since having a `clear` function floating around at module level is likely to be pretty confusing.

### CODE REVIEW

File: __test-helpers__/createApp.js [hunk 1]
```diff
@@ -22,12 +22,13 @@ export const createApp = isWebpack => {
   const Component2 = create('component2')
   const Component3 = create('component3')
 
-  return props =>
-    (<div>
+  return props => (
```
Comment (threehams) [inline __test-helpers__/createApp.js]: `prettier` reformatted these. (I went through some line-ending issues on this PR, so some files likely got touched accidentally.)

File: __tests__/requireUniversalModule.js [hunk 2]
```diff
@@ -244,8 +245,7 @@ describe('requireAsync: requires module asynchronously on the client, returning
 
 describe('addModule: add moduleId and chunkName for SSR flushing', () => {
   it('babel', () => {
-    flushModuleIds() // insure sets are empty:
-    flushChunkNames()
+    clearChunks()
```
Comment (threehams) [inline __tests__/requireUniversalModule.js]: Do you want more explicit tests for `clearChunks()`? I'm not sure if these fully exercise the function.

Comment (faceyspacey) [reply __tests__/requireUniversalModule.js]: na, it's good. I'm not too worried.

**Comment by faceyspacey** [comment]
excellent. Perfect readme! Thanks for this.

**Pull request merged**